initialize.py

```diff
@@ -284,10 +284,10 @@ def initialize_django(adminuser, adminpw):
 def main():
     help_msg = "LNDg Initializer"
     parser = argparse.ArgumentParser(description = help_msg)
-    parser.add_argument('-ip', '--nodeip',help = 'IP that will be used to access the LNDg page', default='*')
+    parser.add_argument('-aip', '--allowed-ip',help = 'Allowed IP to access the LNDg page (IP of the device you want to access LNDg from)', default='*')
```

Comment:
This is not for allowing remote addresses, these are the addresses/hosts that lndg will allow the site to be served over.

Reply:
Ah I see!